samples/user-interface/haptics/README.md

```diff
@@ -8,26 +8,20 @@ Haptics can enrich your app experience by attracting
 the user's attention when it’s required, providing useful feedback to their interactions, and
 augmenting audio-visual content with tactile sensations.
 
-The sample is built on top of [Jetpack Compose](https://developer.android.com/jetpack/compose) and requires a recent version of [Android Studio](https://developer.android.com/studio). 
-
-It uses the Gradle build system. To build this project, clone the repo and use the "gradlew build" command or use "Import Project" in Android Studio (***File > New > Import Project***).
-
 > Important Note: Many of the primitives used in this sample are not yet widely available
 on Android phones.The app will notify you if the device you are using does not
 support the example as intended. 
 
-# Home screen & examples
-
-## Home screen
+## Vibration effects
```

Comment:
Since we removed the home screen description here, this seems a bit redundant since it basically describes the samples of this app. Maybe we can remove this headline altogether? The paragraphs here seem to fit the main summary block naturally.

Reply:
Sorry, my bad, this is a separate sample page with buttons for each vibration effect available. Please feel free to leave it as is.